### ISSUE TITLE
The aria-label attribute of the property grid should be configureable

### ISSUE THREAD

**Issue by tsv2013** [surveyjs/survey-creator#830]
For more details see the https://surveyjs.answerdesk.io/ticket/details/t4338/accessibility-issue-survey-designer-screen-reader-unable-to-configure-toolbox-property thread
